ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -129,3 +127,10 @@ def disable():
     if getattr(Session.request, "opentelemetry_ext_requests_applied", False):
```

Comment:
should `disable_session` be used here as well?

Reply:
Those are slightly different, this one modifies `Session.request`, i.e, the method in the whole class, while disable_session only modifies the request method on that object.